aspnetcore/performance/response-compression/sample/ResponseCompressionSample.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">

```

Comment:
I missed one (big) thing (sort of) ... `net451` ... I'll update for that.

Silly me ... trying to **KILL** the old .NET Framework. Why in the world would I ever want to do that. :wink: